troubleshooting/debugging.md

```diff
@@ -593,6 +595,109 @@ To work around this issue, we suggest using the following option so that system
+    b. Add an `@INCLUDE` statement for your custom config file in the same working directory. In this case, its `@INCLDUE extra.conf`. 
+    c. [Optional for most repros] Add the null output for TCP healthcheck logs to the `fluent-bit.conf` from [here](https://github.com/aws-samples/amazon-ecs-firelens-under-the-hood/blob/mainline/generated-configs/fluent-bit/generated_by_firelens.conf#L31). Please note that the [TCP health check is no longer recommended](https://github.com/aws-samples/amazon-ecs-firelens-examples/tree/mainline/examples/fluent-bit/health-check).
+    d. Create an `[OUTPUT]` configuration based on the `logConfiguration` `options` map for each container in the task definition. The `options` simply become key value pairs in the `[OUTPUT]` configuration.
+    e. Your configuration is now ready to test! If you have a more complicated custom configuration file, you may need to perform additional steps like adding parser files to your working directly, and updating the import paths for them in the Fluent Bit `[SERVICE]` section if your custom config had that. 
+5. Next you need to run Fluent Bit. You can use the following command, which assumes you have an AWS credentials file in `$HOME/.aws` which you want to use for credentials. Run this command in the same working directory that you put your config files in. You may need to customize it to run a different image, mount a directory to dump core files (see our core file tutorial) or add environment variables. 
+
+```
+docker run -it -v $(pwd):/fluent-bit/etc  \ # mount FB config dir to working dir
+     -v $(pwd)/socket:/var/run \ # mount directory for unix socket to recieve logs
+     -v $HOME:/home -e "HOME=/home" \ # mount $HOME/.aws for credentials
+     public.ecr.aws/aws-observability/aws-for-fluent-bit:{TAG_TO_TEST}
+```
+6. Run your application container. You may or may not need to create a mock application container. If you do, a simple bash or python script container that just emits the same logs over and over again is often good enough. We have some [examples in our integ tests](https://github.com/aws/aws-for-fluent-bit/tree/mainline/integ/logger). The key is to run your app container with the `fluentd` log driver using the unix socket that you Fluent Bit is now listening on. 
+
+```
+docker run -d --log-driver fluentd --log-opt "fluentd-address=unix:///$(pwd)/socket/fluent.sock" {logger image}
+```
+
+You now have a local setup that perfectly mimicks FireLens!
+
+For reference, for this example, here is what the `fluent-bit.conf` should look like:
+
+```
+[INPUT]
+    Name forward
+    unix_path /var/run/fluent.sock
+
+[INPUT]
+    Name forward
+    Listen 0.0.0.0
+    Port 24224
+
+[INPUT]
+    Name tcp
+    Tag firelens-healthcheck
+    Listen 127.0.0.1
+    Port 8877
+
+[FILTER]
+    Name   grep
+    Match app-firelens*
+    Regex  log [Ee]rror
```

Comment:
Does this only include logs with the text Error or error in the "log" field?

Reply:
oops/thanks... I dunno why I made this mistake. I swear I remember when I was working on it that I said to myself, need to add a note about not copying just this one filter 🫠